docs/release_notes/v0.17.4.md

```diff
@@ -0,0 +1,34 @@
+# New in 0.17.4 (2021-09-19)
+
+## Changes
+- Update to PyTorch Lightning 1.7 [#1622].
+- Update publication links in `README` [#1667].
+- Change to `super().__init__()` [#1671].
+- Add linkcode, hoverxref, and grid to docs [#1678].
+
+## Bug Fixes
+- Fix likelihood parameters in `RNASeqMixin` [#1618].
```

Comment:
I know you weren't here for most of these, but I would add specifically what this fixes, fix `get...` when gene likelihood is not zinb (as an example)

Reply:
Got it! I wasn't sure how much detail to include here - I'll take a closer look at the diffs from now on then.